From c296589f5a847a00561e25cdfa485b7d53bf4ccd Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Fri, 8 Oct 2021 13:28:46 +0000
Subject: [PATCH 0975/1000] MANTIS 0061647 extension: 0061647:
 Conveyor-Component: new mode for conveyor: maintenance, forceBag, off
 CovneyorDev.idl V1.20 add error type       ERR_MAINTENANCE_MODE,         /**<
 conveyor is manually switched off for maintenance purpose */      
 ERR_FORCEBAG_MODE,            /**< floorwalker can inject bag without checks
 */       ERR_OFF_MODE                  /**< conveyor is manually switched
 offline */

git-svn-id: svn://localhost/SelfServiceCommon/trunk@1677 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl b/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
index bb3c8e45..747a2375 100644
--- a/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/ConveyorDev.idl
@@ -39,7 +39,7 @@
 **/
 module conveyordev
 {
-   const string interface_version = "1.19";
+   const string interface_version = "1.20";
 /*
 using DeviceComponents::indicationListener and DeviceComponents::Generic from mdci.idl
   RC indicationListener::indication(in any ind);
@@ -197,7 +197,10 @@ using DeviceComponents::indicationListener and DeviceComponents::Generic from md
       ERR_VOLUME_NOT_DETERMINABLE,  /**< The SBD system cannot perform a volumetric analysis of the bag */
       ERR_OVERFLOW_TUB,             /**< Detected bag overflows the sides of the tub */
       ERR_STAFF_INTERFERENCE,       /**< Temporary staff interference suspends baggage handling */
-      ERR_INITIALIZING              /**< In transition to a stable state */
+      ERR_INITIALIZING,             /**< In transition to a stable state */
+      ERR_MAINTENANCE_MODE,         /**< conveyor is manually switched off for maintenance purpose */
+      ERR_FORCEBAG_MODE,            /**< floorwalker can inject bag without checks */
+      ERR_OFF_MODE                  /**< conveyor is manually switched offline */
    };
 
 
-- 
2.41.0.windows.1

